save/21340.0.txt

```diff
@@ -0,0 +1,66 @@
+@0
+MOV UP, DOWN
+
+
+@1
+
+
+
+@2
+
+
+
+@3
+
+
+
+@4
+MOV UP, DOWN
+
+
+@5
+MOV UP, RIGHT
+@6
+START:MOV LEFT,ACC
+
+MOV ACC, RIGHT
+
+JGZ P
+
+#checks to see if the value is greater than 0 and then outputs 1 
+MOV 0, DOWN
+
+#if the value is not greater than 0 we move 1 down and output it
+JMP START
+
+P:MOV 1, DOWN
+
+
+@7
+START:MOV LEFT,ACC
+
+MOV ACC, RIGHT
+
+JEZ P
+
+#checks to see if the value is equal to 0 and then outputs 1
+MOV 0, DOWN
+
+#if the value is not greater than 0 we move 1 down and output it
+JMP START
+
+P: MOV 1, DOWN
+
+
+@8
+START:MOV LEFT,ACC
+
+JLZ P
+
+#checks to see if the value is less than 0 and then outputs 1
+MOV 0, DOWN
+
+#if the value is not greater than 0 we move 1 down and output it
```

Comment:
Decent comments here. Would be clearer if the comment was either above or to the right of the line of code it refers to, though.